server/src/weaverbird/backends/pypika_translator/translators/snowflake.py

```diff
@@ -1,9 +1,10 @@
 from typing import Any, TypeVar
 
-from pypika.dialects import SnowflakeQueryBuilder
+from pypika.dialects import QueryBuilder, SnowflakeQueryBuilder
 from pypika.enums import Dialects
-from pypika.queries import Query
+from pypika.queries import AliasedQuery, Query, Selectable
```

Comment:
```suggestion
from pypika.queries import Query, Selectable
```

Just following @lukapeschke review !